app/src/components/ArticleListItem.js

```diff
@@ -23,12 +23,13 @@ class ArticleListItem extends React.Component {
 			>
 				<div className="left">
 					<Img
-						height="100"
-						width="100"
+						width="75"
+						height="75"
 						src={[
 							this.props.images.og,
 							getPlaceholderImageURL(this.props._id),
```

Comment:
Are we storing the image URL in the ID field somewhere? This doesn't seem right but I could be wrong :)

Reply:
no, we don't. the id is used as a random input to select placeholder image, in case of missing imageURL!

Reply:
Ah. Can we make it fall back to the Winds icon instead? It will look much better than a random UUID :)

Reply:
I think using fallback images listed here (https://github.com/GetStream/Winds/tree/master/app/src/images/cards) is by design. The weird part here is the function used to select the random fallback image. it could be much simpler.

Reply:
Okay, I'll let you do what you feel is best. :)